app/src/views/TermsAndConditions/index.tsx

```diff
@@ -156,22 +176,22 @@ export function Component() {
                             <ul>
                                 <li>
                                     {strings.userRegistrationDescriptionList}
-                                    <p>
+                                    <div>
```

Comment:
Do we need div here?

Reply:
Yes, to ensure the paragraph break, we cannot use `p` here because of the html validation rule that `div` cannot appear as children of `p`